city_scrapers/spiders/alle_county.py

```diff
@@ -0,0 +1,170 @@
+        the `Open Civic Data event standard
+        <http://docs.opencivicdata.org/en/latest/data/event.html>`.
+
+        Change the `_parse_id`, `_parse_name`, etc methods
+        to fit your scraping needs.
+        """
+        events = self._make_legistar_call()
+        return self._parse_events(events)
+
+    def _make_legistar_call(self, since=None):
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+        les = LegistarEventsScraper(requests_per_minute=0)
+        les.EVENTSPAGE = self.START_URL + '/Calendar.aspx'
+        les.BASE_URL = self.START_URL
+        if not since:
+            since = datetime.today().year
+        return les.events(since=since)
+
+    def _parse_events(self, events):
+        for item, _ in events:
+            start_time = self._parse_start(item)
+            end_time = self._parse_end(item)
+            data = {
+                'timezone': self.timezone,
+                '_type': 'event',
+                'name': self._parse_name(item),
+                'event_description': '',
+                'all_day': self._parse_all_day(item),
+                'classification': self._parse_classification(item),
+                # _generate_id takes `start` instead of start_time.
+                # `start` is only used internally.
+                'start': start_time,
+                'start_time': start_time,
+                'end_time': end_time,
+                'end': end_time,
+                'location': self._parse_location(item),
+                'documents': self._parse_documents(item),
+                'sources': self._parse_sources(item),
+            }
+            data['id'] = self._generate_id(data)
+            data['status'] = self._generate_status(data, '')
+            yield data
+
+    def _parse_location(self, item):
+        """
+        Parse or generate location.
+        """
+        Room = item.get('Meeting Location', None)
+        if Room:
+            Room = Room + ','
+
+        return {
+            'address': "{Room} {Location}"
+            .format(Room=Room, Location=('436 Grant Street, '
+                                         'Pittsburgh, PA 15219')),
+            'name': '',
+            'neighborhood': ''
+        }
+
+    def _parse_all_day(self, item):
+        """
+        Parse or generate all-day status. Defaults to false.
+        """
+        return False
+
+    def _parse_name(self, item):
+        """
+        Parse or generate event name.
+        """
+        return item['Name']['label']
+
+    def _parse_classification(self, item):
+        """
+        Differentiate board meetings from public hearings.
+        """
+        if 'hearing' in item['Meeting Location'].lower():
+            return FORUM
+        return BOARD
+
+    def _parse_documents(self, item):
+        """
+        Parse or generate documents.
+        """
+        documents = []
+
+        # Add meetings details if available
```

Comment:
I think this is based off another scraper, so it's something we'll need to clean up on our end, but the Meeting Details is more of a page than a document. What we really want here are the Minutes and the Agenda if available. It might also be worth checking for Video, since that's provided even though it doesn't look like it's ever been available (in which case we should just ignore it)

Reply:
Agenda is available but not Minutes.

Reply:
I'm seeing several published minutes for some meetings in the past, [here's one example](https://alleghenycounty.legistar.com/MeetingDetail.aspx?ID=622063&GUID=E611BFA0-0E2F-4B45-9DEB-3489E5F37790&Options=info&Search=). We'll need to check if it's there, and only add it if it's present, but it does look like several meetings have them

Reply:
I am not familiar with these records. Are you talking about url link associated with `Published Mintues:`??? `legistar` is not grabbing url link from `Minutes`.

Reply:
I see.  I got it. Do you also want me to change `note` here to `Meeting Details [Minutes]`??
```
details = item['Minutes']
        if type(details) == dict:
            documents.append({
                'note': 'Meeting Details'
```

Or just leave it as `'note': 'Meeting Details'`

Reply:
Yeah, change it to Minutes and that should fix it